src/java/org/apache/cassandra/db/commitlog/SimpleCachedBufferPool.java

```diff
@@ -72,6 +74,7 @@ public SimpleCachedBufferPool(int maxBufferPoolSize, int bufferSize)
     {
```

Comment:
`preferredReusableBufferType` above should no longer be necessary.

Reply:
That I tried and then you need to do a `BufferType.typeOf()`... So it's an extra operation vs having that stored in a variable.